ores/wsgi/util.py

```diff
@@ -82,3 +84,17 @@ def parse_injection(request, rev_id):
         return {rev_id: cache} if len(cache) > 0 else None
     except Exception as e:
         raise CacheParsingError(e)
+
+
+def nocache(route):
+    @wraps(route)
+    def no_cache(*args, **kwargs):
+        response = make_response(route(*args, **kwargs))
+        response.headers['Cache-Control'] = \
+            "no-store, no-cache, must-revalidate, post-check=0, " + \
+            "pre-check=0, max-age=0"
+        response.headers['Pragma'] = 'no-cache'
+        response.headers['Expires'] = '-1'
```

Comment:
https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html says Expires needs to be in RFC 1123 days format. And -1 is not that. Either set it to epoch (Jan 1st 1970) or remove it